test/commands/add/sas-add.spec.js

```diff
@@ -0,0 +1,74 @@
+import dotenv from 'dotenv'
```

Comment:
pls rename this file to `sasjs-add.spec.js` tp reflect the exact command it is covering